src/api/composition/CompositionCollection.js

```diff
@@ -203,50 +219,39 @@ define([
                 this.mutables[keyString] = child;
             }
 
-            this.emit('add', child);
+            this.#emit('add', child);
         }
-    };
-
+    }
     /**
      * Load the domain objects in this composition.
      *
-     * @returns {Promise.<Array.<module:openmct.DomainObject>>} a promise for
+     * @param {any} abortSignal
```

Comment:
[`abortSignal` is a builtin type](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal), are we able to just reference it without having to define it?

Reply:
Yes! Good catch.